File: GraffiTab-iOS/CustomUI/Cell/Comments/Autocomplete/AutocompleteHashCell.m
//
//  AutocompleteHashCell.m
//  GraffiTab-iOS
//
//  Created by Georgi Christov on 03/04/2015.
//  Copyright (c) 2015 GraffiTab. All rights reserved.
//

#import "AutocompleteHashCell.h"

@implementation AutocompleteHashCell

+ (CGFloat)height {
    return 53;
}

- (void)setSelected:(BOOL)selected {
    UIView *bgColorView = [[UIView alloc] init];
    bgColorView.backgroundColor = [UIColor colorWithHexString:@"#F5EC89" alpha:0.3];
    [self setSelectedBackgroundView:bgColorView];
}

- (void)setItem:(NSString *)item {
    _item = item;
    
    self.hashLabel.text = item;
}

@end

File: GraffiTab/CustomUI/Cells/Comments/Autocomplete/AutocompleteHashCell.swift
//
//  AutocompleteHashCell.swift
//  GraffiTab
//
//  Created by Georgi Christov on 11/05/2016.
//  Copyright © 2016 GraffiTab. All rights reserved.
//

import UIKit

class AutocompleteHashCell: UITableViewCell {

    @IBOutlet weak var nameField: UILabel!
    
    var item: String? {
        didSet {
            setItem()
        }
    }
    
    class func reusableIdentifier() -> String {
        return "AutocompleteHashCell"
    }
    
    func setItem() {
        // Setup labels.
        self.nameField.text = "#\(item!)"
    }
}
